packages/jasmine/src/adapter/JasmineAdapter.ts

```diff
@@ -80,6 +80,13 @@ export class JasmineAdapter implements TestRunnerAdapter {
         ));
 
         this.runner.addReporter(reporter(this.runner.jasmine));
+
+        if (this.config.additionalReporters){
+            const additionalReporters = this.config.additionalReporters
+            console.log(`${additionalReporters.length} Additional reporter(s) found. Going to configure them...`)
+            additionalReporters.forEach(additionalReporter =>  this.runner.addReporter(additionalReporter));
```

Comment:
Would you mind adding a unit test to make sure that whenever `config.reporters` is specified, we add those reporters? Have a look at [`packages/jasmine/spec/adapter/JasmineAdapter.spec.ts`](https://github.com/serenity-js/serenity-js/blob/master/packages/jasmine/spec/adapter/JasmineAdapter.spec.ts)

Reply:
I added a couple of tests: 1 for default reporter and another for an additional reporters.
I hope they are ok, long time without doing unit tests :)